### PR TITLE
Rov 32

### PR DESCRIPTION
**Pull request by Varuno8**

### CODE REVIEW

File: signup.html [hunk 1]
```diff
@@ -0,0 +1,234 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Two Halves Webpage</title>
+    <style>
+      body {
```
Comment (rachittyagi) [inline signup.html:8]: move all the style to css/style.css

File: signup.html [hunk 2]
```diff
@@ -0,0 +1,234 @@
...
+          <input
+            type="password"
+            id="passwordField"
+            class="input-field"
+            placeholder="Password"
+          />
+          <button
+            id="toggleButton"
+            onclick="togglePassword()"
+            style="border: none; background: transparent; cursor: pointer"
```
Comment (rachittyagi) [inline signup.html:169]: remove the inline style and add a class for it

File: signup.html [hunk 3]
```diff
@@ -0,0 +1,234 @@
...
+          >
+            <img
+              src="pngwing.com (1) copy.png"
```
Comment (rachittyagi) [inline signup.html:172]: move all the images to assets/img folder and add reference from there

File: signup.html [hunk 4]
```diff
@@ -0,0 +1,234 @@
...
+          });
+        </script>
+        <button class="signup-button" style="margin-top: 40px">Sign Up</button>
+        <p style="color: black; font-size: 15px; margin-top: 15px">
+          Already have an account?
+          <a
+            href="#"
+            style="color: blue; text-decoration: underline; font-size: 15px"
+            >Login</a
+          >&gt
```
Comment (rachittyagi) [inline signup.html:201]: remove &gt

File: signup.html [hunk 5]
```diff
@@ -0,0 +1,234 @@
...
+      <div class="right-content" style="margin-top: 80px">
+        <h1 style="margin-top: -145px">MOMENTS</h1>
+        <p style="font-size: 33px; margin-top: 50px; padding-top: 30px">
+          "Formal education will make you living;
+          <br />
+          self-education will make you a fortune."
+        </p>
+
+        <blockquote
+          style="
```
Comment (rachittyagi) [inline signup.html:222]: remove inline style and add class for it

File: signup.html [hunk 6]
```diff
@@ -0,0 +1,234 @@
...
+        color: #1bba9b;
+        position: relative;
+        right: -20px;
+        /* margin-top: 60px; */
+      }
+    </style>
+  </head>
+  <body>
+    <div class="half left-half">
+      <img src="photo_2023-10-21 21.01.13.jpeg" alt="Logo" class="logo" />
```
Comment (rachittyagi) [inline signup.html:149]: move the image to assets/img folder and add reference form there

**Review by rachittyagi — CHANGES_REQUESTED** [review]

File: signup.html (hunk 1 above)
Comment (rachittyagi) [reply signup.html:8]: And change the title to Sign Up

**Comment by yaneshtyagi** [comment]
Branch is not ready for merging. Closing this PR as this story should go back to backlog.

**Pull request closed without merging**